_posts/2017-12-30-new-years-rust-a-call-for-community-blogposts.md

```diff
@@ -0,0 +1,71 @@
+# New Year's Rust: A Call for Community Blogposts
+
+Tis the season for people and communities to reflect and set goals- and the Rust team is
+no different. Last week, we published [a blogpost about our accomplishments in 2017](rust-2017),
+and the teams have already begun brainstorming goals for next year. One thing that we know for
+certain is that Rust is commited to extending and improving our community organization and
+outreach- and this post is just the first of what we hope to be many iterations of new kinds of
+community feedback mechanisms.
+
+[rust-2017]: https://blog.rust-lang.org/2017/12/21/rust-in-2017.html
+
+Last year, the Rust team started a new tradition: defining a roadmap of goals for the 
+upcoming year. We leveraged our 
+[RFC process](https://github.com/aturon/rfcs/blob/roadmap-2017/text/0000-roadmap-2017.md)
+to solicit community feedback. While we got a lot of awesome feedback on that RFC, we'd 
+like to try something new **in addition to** the RFC process: a call for community blog posts
+for ideas of what the goals should be.
```

Comment:
Maybe you could mention *why* this additional step is needed, i.e. so it's explicit that the core team isn't dictating the roadmap and then going through the RFC process just "pro forma". I'm not sure if that fits in here (and it certainly would have to be formulated in a more positive way `:)`) but I expect the question "Why change something now?" to pop into people's mind at this point.